tsconfig.base.v8.json

```diff
@@ -0,0 +1,56 @@
+{
+  "compilerOptions": {
+    "moduleResolution": "Node",
+    "pretty": true,
+    "sourceMap": true,
+    "skipLibCheck": true,
+    "forceConsistentCasingInFileNames": true,
+    "typeRoots": ["node_modules/@types", "./typings"],
+    "rootDir": ".",
+    "baseUrl": ".",
+    "paths": {
+      "@fluentui/api-docs": ["packages/api-docs/src/index.ts"],
```

Comment:
this map has been verified on whole monorepo in this PR, https://github.com/microsoft/fluentui/pull/22604/files